app/assets/stylesheets/components/user_header.scss

```diff
@@ -0,0 +1,25 @@
+/* Header Styles */
+
+.user-header.navbar-default {
+  background-color: $orange;
+
+  .navbar-nav > li > a {
+    color: $white80;
+  }
+
+  .navbar-nav > li > a:hover, a:focus, a:active {
+    color: $white40;
+  }
+
+  .link-container.navbar-right {
+    padding-right: 100px;
+  }
+}
+
+.header__logo {
```

Comment:
.header__logo is already defined in header.scss so you don't have to call it here. or you can name it something else